CHANGELOG

```diff
@@ -1,16 +1,16 @@
-Version 2.3.1, 2022-06-19
--------------------------
+# CHANGELOG
```

Comment:
If going to use markdown syntax throughout the file, why not add the extension, so that GH will render it more nicely when viewing it?

Reply:
I was thinking about it. What is your opinion on that? Should we change the file to a markdown file (+ syntax changes; maybe also linking all of the PRs)?

Reply:
I'm not sure I get fully what you mean on the first sentence. You've already modified the file to be markdown in the syntax of it, you just didn't add the file extension. I see no reason for the file to not be markdown, especially given that is necessary to include it in the docs.

Linking to PRs is definitely nice.

Reply:
> I'm not sure I get fully what you mean on the first sentence.

When I opened this PR, I wasn't sure if I really want to merge it. I wanted to see your and other peoples reaction. From your comment I take that you like it (with some adjustments) :-)

Reply:
I should have formally asked for a review :sweat_smile:

Reply:
Yup, I like markdown for things, you can get it nicer rendered, and do stuff like have links to relevant issues / PRs as part of it. :+1:

Re-reading my comments, I apologize that they were unnecessarily curt, I could have phrased what I meant in a better way.

Reply:
Nice!

All good - I didn't mind :heart:  And I think I might be court in several cases as well :sweat_smile: